packages/plugin-core/src/workspace.ts

```diff
@@ -304,6 +304,10 @@ export class DendronExtension implements IDendronExtension {
     return false;
   }
 
+  isActiveAndIsDendronNote(_fpath: string): boolean {
+    throw new Error("not implemented");
```

Comment:
So to follow through - let's move the logic from the current impl in `ExtensionProvider` to here, and then replace the impl in ExtensionProvider with 

```typescript
  static isActiveAndIsDendronNote(fpath: string) {
    return ExtensionProvider.getExtension().isActiveAndIsDendronNote(fPath);
  }
```